programs/test/benchmark.c

```diff
@@ -358,6 +402,12 @@ int main( int argc, char *argv[] )
                 todo.ecdsa = 1;
             else if( strcmp( argv[i], "ecdh" ) == 0 )
                 todo.ecdh = 1;
+            else if( strcmp( argv[i], "buffer-16" ) == 0 )
```

Comment:
It is better to receive a number and set the size, this way we will not be limited to only 3 sizes. 
The buffers can be smaller then 1024 if we think it is enough as a max buff size

Reply:
Fixed :)